### PR TITLE
Add pull_request_template.md

### PR DESCRIPTION
**Pull request by BatoolMM**
I've seen few people add their content to highlight sections or un-related sections when submitting their PR. 
In order to make the process easier, I'm suggesting a very small template PR!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Adding new content to R Weekly issue
+
+Please include a one-sentence summary of the new content you would like to be added to our R Weekly issue.
+
+## Type of Content
+
+**Please select to which section(s) your post belongs!**
+
+<!--Please don't add anything to the Highlight section, R Weekly members vote for the content of this section weekly-->
```
Comment (jonocarroll) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]:
```suggestion
<!--Please don't add anything to the Highlight section, R Weekly editors vote for the content of this section weekly--> 
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Resources - long posts, websites, books, slides, list, cheat sheets, or other learning resources in general that are more officially aggregated as a guide material
+- [ ] Jobs - R Jobs
+- [ ] New Packages and Tools - New packages and tools that have been created or published in the past two weeks.
+- [ ] Updated Packages - New releases of tools and packages for R
+- [ ] Call for Participation - New R groups, communities or competitions here. 
+- [ ] Upcoming Events - Interesting R-related events or call for Participation section.
+
+### I'd like to propose an Image from my new content!
+
+<!--Although you are very welcome to suggest an image, please bear in mind that images are added and changed by the editor in charge of the issue so we can guarantee that the image will be added-->
```
Comment (jonocarroll) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]:
```suggestion
<!--Although you are very welcome to suggest an image, please bear in mind that images are added and changed by the editor in charge of the issue so we can't guarantee that your image will be added--> 
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] New Packages and Tools - New packages and tools that have been created or published in the past two weeks.
+- [ ] Updated Packages - New releases of tools and packages for R
+- [ ] Call for Participation - New R groups, communities or competitions here. 
+- [ ] Upcoming Events - Interesting R-related events or call for Participation section.
+
+### I'd like to propose an Image from my new content!
+
+<!--Although you are very welcome to suggest an image, please bear in mind that images are added and changed by the editor in charge of the issue so we can guarantee that the image will be added--> 
+
+Please feel free to suggest an image from your content to be added in the next issue
```
Comment (jonocarroll) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: We could maybe add the suggestion of max width 600px

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- [ ] Yes, I proposed an image and resized it
+- [ ] No
+
+# Checklist:
+
+- [ ] My content is R-related 
+- [ ] All images suggested are re-sized before has been added to the PR
+- [ ] I've submitted my content between Monday-Friday to allow for it to be voted by R Weekly members for highlights
```
Comment (jonocarroll) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]:
```suggestion
- [ ] I've submitted my content between Monday-Friday to allow for it to be voted by R Weekly editors for highlights
```

**Comment by jonocarroll** [comment]
This looks great! Thanks for taking the initiative to put this together.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 3 above)
Comment (BatoolMM) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Thank you, Jonathan, I changed the wording. Is this alright?

**Review by jonocarroll — APPROVED** [review]
Looks great! Thanks again.

**Comment by BatoolMM** [comment]
Thank you for taking the time to review it, I'll merge it now!

**Pull request merged**